### DISCUSSION TITLE
home

### DISCUSSION THREAD

**Discussion by giscus** [xy-241/CS-Notes#2] [General]
# index

Homepage is still under development. You can navigate around the notes using the explorer (not yet available on mobile devices) on the left side. Please feel free to provide feedback regarding the accuracy of the notes, etc.

https://notes.yxy.ninja/

**Comment by xy-241** [comment]
Hello!

**Comment by OliverTeo288** [comment]
hi xinyang

**Reply by xy-241** [reply]
Good Morning Sir 🫡!

**Reply by OliverTeo288** [reply]
wah sick got email notification somemore

**Comment by doeixd** [comment]
Thanks for publishing these notes.

**Reply by xy-241** [reply]
Hi, thanks for the comment. Hopefully, the notes are useful to you. Let me know if there is anything confusing or incorrect. It is a great opportunity for me to learn too. :)

**Comment by aemooooon** [comment]
Kia ora

**Reply by xy-241** [reply]
Kia ora, great to see you here :)

**Comment by kumailr7** [comment]
Really loved the site , taking a greater inspiration from you , keep it up !!

**Reply by xy-241** [reply]
Thanks for the kind words, really appreciate it 😊!

Can I trouble you to remove my analytics codes from your site? So your site usage metrics aren't pushed to me, thanks! 

[analytics code 1](https://github.com/kumailr7/Opscatalyst/blob/v4/quartz/plugins/emitters/componentResources.ts#L100-L117)
[analytics code 2](https://github.com/kumailr7/Opscatalyst/blob/v4/quartz.config.ts#L15-L16)
[analytics code 3](https://github.com/kumailr7/Opscatalyst/blob/v4/quartz/components/Head.tsx#L23-L49)

**Comment by David7ce** [comment]
Nice and beautiful wiki made with Quartz and Obsidian. I have forked to a [list of good PKM Vaults on GItHub](https://github.com/orgs/obsidian-pkm-vault/repositories) 👍

**Reply by xy-241** [reply]
Thanks for the compliments & forking my repo 😊!

**Comment by Grazinascito** [comment]
wow🤩

**Comment by GuisandeGolfer** [comment]
Hey bro! such an awesome site. Aspiring to reach your content quality on my site.

Your site is a great resource for my CS job hunt! I would love to see more content on career tips.

**Reply by xy-241** [reply]
Hey Diego,

Thanks for the kind words! Glad my site helped you out.  If you spot any mistakes or anything that's not clear, just let me know! I appreciate it.

And that's a really cool idea!  I just finished up my job hunt, so once finals are over, I'm gonna take some time to think about it all and share some career tips. Stay tuned! 😄

**Comment by fred1357944** [comment]
Wonder how do you import this commet funtion on this website
could you teach me?

**Reply by xy-241** [reply]
Hi, sorry for the late reply! I created this comment plugin for my site. You can [fork my repo](https://github.com/xy-241/CS-Notes/fork) and modify the [comment settings](https://github.com/xy-241/CS-Notes/blob/v4/quartz/components/Comments.tsx#L9-L24) with your own GitHub comment settings, which can be obtained from [giscus](https://giscus.app/). 

Remember to remove my [analytics code 1](https://github.com/xy-241/CS-Notes/blob/v4/quartz/components/Head.tsx#L189-L215), [analytics code 2](https://github.com/xy-241/CS-Notes/blob/v4/quartz/plugins/emitters/componentResources.ts#L100-L116), and [analytics code 3](https://github.com/xy-241/CS-Notes/blob/v4/quartz.config.ts#L16-L17) so your site doesn't send analytics data to mine.

Happy hacking!
